styleguide/global.requires.js

```diff
@@ -157,8 +159,8 @@ window.emojiAddRecent = emojiAddRecent
 window.getCurrentSkinTone = getCurrentSkinTone
 window.setCurrentSkinTone = setCurrentSkinTone
 window.usernameToColor = usernameToColor
-// Exported composables
-window.useIsDarkTheme = useIsDarkTheme
```

Comment:
You didn't like `useIsDarkTheme` feature? 😥

Reply:
https://github.com/nextcloud-libraries/nextcloud-vue/commit/430f0f395e1cda497bfb7e80daa34b58b02213da#r149050926

But unrelated for this PR